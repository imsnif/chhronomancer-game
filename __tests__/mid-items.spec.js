import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

test('Mid-Items component => renders correctly', () => {
  const MidItems = require('../components/mid-items').default
  const items = [
    {fill: true, name: 'lock', bright: true}
  ]
  const tree = renderer.create(
    <MidItems items={items} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
