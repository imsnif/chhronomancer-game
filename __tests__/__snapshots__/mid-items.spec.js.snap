// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MidItems (default props) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "borderColor": "purple",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "black",
        "borderColor": "brown",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
</View>
`;

exports[`MidItems (with brown item) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "borderColor": "purple",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "brown",
        "borderColor": "brown",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
</View>
`;

exports[`MidItems (with purple and brown items) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "purple",
        "borderColor": "purple",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "brown",
        "borderColor": "brown",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
</View>
`;

exports[`MidItems (with purple item) => renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "purple",
        "borderColor": "purple",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "black",
        "borderColor": "brown",
        "borderWidth": 1,
        "height": 9,
        "margin": 4,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 9,
      }
    }
  />
</View>
`;
